### PR TITLE
Felinids are now able to laugh + Two new emotes

### PR DESCRIPTION
**Pull request by Hardly3D**
## About The Pull Request
Adds *snap (snaps their finger), *tilt (tilts their head to the side) and felinids are now able to laugh audibly, they use the same laugh sounds as humans.
Emotes are ported from:
https://github.com/BeeStation/BeeStation-Hornet/pull/2854
https://github.com/BeeStation/BeeStation-Hornet/pull/1208
https://github.com/tgstation/tgstation/pull/59585

## Why It's Good For The Game
Makes more sense for felinids to be able to laugh as they are just mutated humans or something
Also RP

## Changelog
:cl:Hardly
add: Added two new emotes: *snap (snaps their finger) and *tilt (tilts their head to the side)
tweak: Felinids can now laugh audibly
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
cool

**Comment by Kapu1178** [comment]
I have some minor code gripes with this PR but I’m at work so DNM pls.

File: code/modules/mob/living/emote.dm [hunk 1]
```diff
@@ -218,7 +218,8 @@
 /datum/emote/living/laugh/get_sound(mob/living/user)
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		if(H.dna.species.id == "human" && (!H.mind || !H.mind.miming))
+		var/human_laugh = ishumanbasic(H) || isfelinid(H)
+		if(human_laugh && (!H.mind || !H.mind.miming))
```
Comment (Kapu1178) [inline code/modules/mob/living/emote.dm:222]: ishumanbasic() returns TRUE for felinids already,
also, this code is bad and snowflakey.

add a `var/list/laughs` to `/datum/species`. then have something like
`if(H.dna.species.laughs)`

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

**Comment by AtlasRules** [comment]
thank you for not making it that annoying fucking citadel nyahaha laugh

**Comment by Ms-Mee** [comment]
> thank you for not making it that annoying fucking citadel nyahaha laugh 

Time to make a PR :joker:

File: code/modules/mob/living/emote.dm (hunk 1 above)
Comment (Hardly3D) [reply code/modules/mob/living/emote.dm:222]: I've done as you suggested and laugh doesn't play a sound anymore, am I supposed to assign something to the list?

Comment (Kapu1178) [reply code/modules/mob/living/emote.dm:222]: > 

You're supposed to populate the list with file paths to each laugh, then you can modify the code to `pick()` from the species laugh list

**Pull request closed without merging**